run_job_mix.sh

```diff
@@ -413,8 +420,9 @@ setup_expr()
     ipcrm --all=msg || :
 }
 
+cd ~/llama2-expr
```

Comment:
Please remove this and make it similar to:
https://github.com/aravindhank11/llama2-expr/blob/ak-live-migration/run_job_mix.sh#L412-L414

Git clone need not be in the home directory always.